docs/source/tutorials/traits_ui_scientific_app.rst

```diff
@@ -822,9 +846,7 @@ physics experiment (Bose-Einstein condensation), at the university of
 Toronto.
 
 The reason I am providing this code is to give an example to study of how
-a full-blown application can be built. This code can be found in the
-`tutorial's zip file <http://gael-varoquaux.info/computers/traits_tutorial/traits_tutorial.zip>`_
-(it is the file `application.py`).
```

Comment:
personally, if possible, i prefer adding links to the archived version of the page - https://web.archive.org/web/20151208222136/http://www.gael-varoquaux.info/computers/traits_tutorial

Reply:
sounds good, I will link that page.  Unfortunately I could not find the actual code files through that page.  Once there, the link that says 'The zip archive with all the code files' yields a blank page.

Reply:
> Once there, the link that says 'The zip archive with all the code files' yields a blank page.

This should yield a blank page because this points to the zip file. Clicking on that link should trigger a download (it did for me).

But pointing to the main page is probably better.